ureport/polls/models.py

```diff
@@ -347,7 +347,7 @@ def get_results(self, segment=None):
         if cached_value:
             return cached_value['results']
 
-        if segment and segment.get('location', "") == "District":
+        if segment and segment.get('location', "") in ("District", 'Ward'):
```

Comment:
i think that it would be nicer to use always the same style for strings here. either use always single quotes or always double quotes on the same line